packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,631 @@
+        {
+          name: 'Bob',
+          url: 'javascript:bob-is-cool' // jshint ignore:line
+        },
+        {
+          name: 'James',
+          url: 'vbscript:james-is-cool' // jshint ignore:line
+        },
+        {
+          name: 'Richard',
+          url: 'javascript:richard-is-cool' // jshint ignore:line
+        }
+      ])
+    });
+
+    this.assertHTML(strip`
+      <ul>
+        <li>
+          <a href="unsafe:javascript:bob-is-cool">Bob</a>
+        </li>
+        <li>
+          <a href="unsafe:vbscript:james-is-cool">James</a>
+        </li>
+        <li>
+          <a href="unsafe:javascript:richard-is-cool">Richard</a>
+        </li>
+      </ul>
+    `);
+  }
+
+  ['@test it should render the current value of a property path on the context with container']() {
+    let owner = buildOwner();
+    this.render(`{{unbound foo}}`, {
+      [OWNER]: owner,
+      foo: 'bleep'
+    });
+
+    this.assertText('bleep');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('bleep');
+
+    this.runTask(() => set(this.context, 'foo', 'bloop'));
+
+    this.assertText('bleep');
+
+    this.runTask(() => set(this.context, 'foo', 'bleep'));
+  }
+
+  ['@htmlbars helper form updates on parent re-render']() {
+    this.render(`{{unbound foo}}`, {
+      foo: 'BORK'
+    });
+
+    this.assertText('BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'OOF'));
+
+    this.assertText('BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('OOF');
+
+    this.runTask(() => set(this.context, 'foo', ''));
+
+    this.assertText('OOF');
+
+    this.runTask(() => set(this.context, 'foo', 'BORK'));
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK');
+  }
+
+  // semantics here is not guaranteed
+  ['@test sexpr form does not update does unless explicitly told to']() {
+    this.registerHelper('capitalize', (args) => args[0].toUpperCase());
+
+    this.render(`{{capitalize (unbound foo)}}`, {
+      foo: 'bork'
+    });
+
+    this.assertText('BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(this.context, 'foo', 'oof');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'blip'));
+
+    this.assertText('BORK');
+
+    this.runTask(() => {
+      set(this.context, 'foo', 'bork');
+      this.rerender();
+    });
+
+    this.assertText('BORK');
+  }
+
+  ['@test sexpr in helper form does not update on parent re-render']() {
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super(...arguments);
+        fooBarInstance = this;
+      },
+      foo: 'bork'
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{capitalize (unbound (doublize foo))}}`
+    });
```

Comment:
Can be rewritten without the component, just use `this.rerender` (which `rerender`s) the harness component